Gemfile

```diff
@@ -15,7 +15,7 @@ end
 #
 
 group :test do
-  gem "cucumber", "~> 3.0"
+  gem "cucumber", "~> 4.1.0"
```

Comment:
Let us relax the constraint on the version so that we need not have to update when Cucumber-4.2 releases in the future.
```suggestion
  gem "cucumber", "~> 4.1"
```

Reply:
This will result in more maintenance work. But I will adapt.

Reply:
It's good that you noticed.